bin/dotd

```diff
@@ -465,6 +467,24 @@ def get_honeybadger_issues
   @logger.info 'gets honeybadger issues'
 end
 
+# Performs a diagnostic self-check to make sure the local environment is
+# properly set up for all standard Dev of the Day operations.
+def perform_self_check
+  puts "Checking access to internal servers"
+  %w(staging test levelbuilder-staging production-console).each do |environment|
+    run_on(environment, "echo 'successfully connected to #{environment}'")
+  end
+
+  puts "Checking access to GitHub"
+  puts "latest staging commit is: #{GitHub.sha('staging', true)}"
+
+  puts "Checking access to Slack"
+  puts "current Developer of the Day is: #{DevelopersTopic.dotd}"
+
+  puts "Checking access to Honeybadger"
+  puts "TODO"
```

Comment:
I'm not sure what we want out of such a check, or if we even really need it. The options that come to me here are:

1. Invoke `Honeybadger.get_recent_issues`; this confirms we can do _all_ desired functionality, but also take some time and generates a lot of output
2. Create a more-targeted method on the `Honeybadger` class we can invoke to just check connectivity without also downloading unnecessary data
3. Don't bother to check Honeybadger access at all in this self-check

I'm leaning toward 3, open to 2, and could be convinced on 1. Open to suggestions!

Reply:
Fwiw, I think honeybadger feels less urgent than the other checks, so would be happy with 3!

Reply:
I'm fine with 3, perhaps with a note similar to your comment?